scripts/languageserver/languageserver.jl

```diff
@@ -6,39 +6,40 @@ if VERSION < v"0.5"
 end
 
 include("dependencies.jl")
-use_and_install_dependencies(["Compat", "JSON", "Lint", "URIParser"])
+#use_and_install_dependencies(["Compat", "JSON", "Lint", "URIParser","JuliaParser"])
+using Compat,JSON,Lint,URIParser,JuliaParser
```

Comment:
Sorry, I didn't mean to merge that and did mean to tell you that I can't get this function to work, I'll have a look to see why later.